src/mockups/VaultV2.sol

```diff
@@ -63,8 +63,8 @@ contract VaultV2 {
     }
 
     struct VaultInfo {
-        uint16 collThres; //2 decimals precision (factor 100)
-        uint8 liqThres; //2 decimals precision (factor 100)
+        uint16 collFactor; //2 decimals precision (factor 100)
```

Comment:
Why do we change the name from collThres to collFactor?
If we do change the name it should be consistent with the rest of the code base:
- Liquidator.sol
- liqThres

Reply:
I mostly see Collateral Factor or Collateral Ratio in this context. Threshold gives a meaning that if it is passed some action will happen, which is true for liquidation, not that true for collateral valuation.